### ISSUE TITLE
object detection - How to repeate the pretrained model accuracy on coco?

### ISSUE THREAD

**Issue by xhzhao** [tensorflow/models#6360]
Our team is working on tensorflow performance optimization for object detections, and we tried to repeat the pretrained model accuracy on coco dataset. AFAIK, there is no guide to show details to repeat the model accuracy.
We tried to use the guide for the Open Image Dataset on this [link](https://github.com/tensorflow/models/blob/master/research/object_detection/g3doc/oid_inference_and_evaluation.md), and it is really suffering and we have to make many hacks, and bad news is that the finally mAP accuracy is not the same as the released data (27 vs 24).

a similar issue found [here](https://github.com/tensorflow/models/issues/6341)
Am i missing something?

![image](https://user-images.githubusercontent.com/17486215/54327811-82587100-4646-11e9-93d8-d9a664de0f80.png)

**Comment by xhzhao** [comment]
we also take a look at the TensorRT solution for tensorflow object detection [here](https://github.com/tensorflow/tensorrt/tree/master/tftrt/examples/object_detection), and we found the mAP accuracy does not match with the released data.

Is there any bug in thensorflow or the model zoo?

**Comment by rootkitchao** [comment]
You can take a look at my issue(https://github.com/tensorflow/models/issues/6278).
In addition, I recently tried to retrain on the MSCOCO17 Train and verified on the MSCOCO17 Val.
Retinanet(Mobilenet V1 FPN) 640*640:
![image](https://user-images.githubusercontent.com/40640909/54381253-4534c300-46c8-11e9-9843-26a51b06bf9c.png)
Retinanet(Mobilenet V2 FPN) 640*640:
![image](https://user-images.githubusercontent.com/40640909/54381307-64335500-46c8-11e9-82b5-11fb3cf9928c.png)

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be an old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
